charts/jenkins/templates/jenkins-master-secondary-ingress.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.master.secondaryingress.enabled }}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: {{ .Values.master.secondaryingress.apiVersion }}
+{{- end }}
+kind: Ingress
+metadata:
+  namespace: {{ template "jenkins.namespace" . }}
+  labels:
+    "app.kubernetes.io/name": '{{ template "jenkins.name" .}}'
+    "helm.sh/chart": "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    "app.kubernetes.io/managed-by": "{{ .Release.Service }}"
+    "app.kubernetes.io/instance": "{{ .Release.Name }}"
+    "app.kubernetes.io/component": "{{ .Values.master.componentName }}"
+{{- if .Values.master.secondaryingress.labels }}
+{{ toYaml .Values.master.secondaryingress.labels | indent 4 }}
+{{- end }}
+{{- if .Values.master.secondaryingress.annotations }}
+  annotations:
+{{ toYaml .Values.master.secondaryingress.annotations | indent 4 }}
+{{- end }}
+  name: {{ template "jenkins.fullname" . }}
+spec:
+  rules:
+    - host: 
+      http:
+        paths:
+        - path: /testpath
```

Comment:
Should this be templated?

Reply:
i honestly don't remember writing that, so i must have started it and forgot